modules/shutter_page/templates/custom_endpoint.json

```diff
@@ -0,0 +1,99 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "profile_name": {
+            "defaultValue": "hmcts-shutter-page-nonprod",
```

Comment:
possibly remove this? defaults can be a pain as sometimes they accidentally get used.

Reply:
Yes, I will remove this bit.